cluster.py

```diff
@@ -0,0 +1,66 @@
+"""Kubernetes Cluster API Class."""
+from kubernetes import client, config
+
+
+class Cluster():
+    """Define Cluster data and methods."""
+
+    def __init__(self, kubeconfig):
+        """Instantiate Cluster object.
+
+        Args:
+            kubeconfig: credential file for cluster
+
+        """
+        self.kubeconfig = kubeconfig
+        self.apps_v1_api = None
+        self.core_v1_api = None
+        self.connect_to_cluster()
+
+    def connect_to_cluster(self):
+        """Connect to the cluster. Set API attributes."""
+        config.load_kube_config(self.kubeconfig)
+        self.apps_v1_api = client.AppsV1Api()
+        self.core_v1_api = client.CoreV1Api()
+
+    def get_pods_for_all_namespaces(self):
```

Comment:
Any thought of a method that gets all pods and places them into a dictionary by namespace? Would that be useful later? Then you could just get all the values from that method for this one.

Reply:
This is a good idea, especially when each component operates within its own namespace, but also when the data needs to be referenced multiple times in the process. I'm going to keep this idea in mind, but implement it once it's needed.